### ISSUE TITLE
Multiple characters

### ISSUE THREAD

**Issue by cmrn** [cmrn/squire#2]
Support switching between multiple characters (likely using a spinner in the action bar)

via [kalebmcc](http://www.reddit.com/r/Pathfinder_RPG/comments/193u1m/i_just_released_the_first_beta_of_an_android_app/c8klor5), [ozymandious](http://www.reddit.com/r/rpg/comments/193t9x/i_just_released_the_first_beta_of_an_android_app/c8klsna) and [theStingraY](http://www.reddit.com/r/rpg/comments/193t9x/i_just_released_the_first_beta_of_an_android_app/c8klvhl) on reddit

**Comment by shane-smith** [comment]
Would also like to see this.

**Comment by honvai** [comment]
Upgrade auto modifiers changer

**Comment by viniciuscsg** [comment]
That is really the first feature to implement.

**Comment by Setherial76** [comment]
yeah, I would really like this feature as well. I love the program but many characters use a second character like a familiar, animal companion, cohort or whatever and you need to be able to track them as well for the program to be really useful.
